environment/src/test/java/com/sequenceiq/environment/credential/service/CredentialServiceTest.java

```diff
@@ -345,7 +346,7 @@ void testInitCodeGrantFlowNoUrl() {
 
     @Test
     void testInitCodeGrantFlowValidationErrorNotSaved() {
-        doThrow(BadRequestException.class).when(credentialValidator).validateCredentialCloudPlatform(anyString(), anyString());
+        doThrow(BadRequestException.class).when(credentialValidator).validateCredentialCloudPlatform(anyString(), anyString(), any(CredentialType.class));
```

Comment:
can we have unit tests for credential type `AUDIT`?

Reply:
done